adhocracy4/maps/widgets.py

```diff
@@ -22,9 +22,9 @@ def render(self, name, value, attrs):
 
         context = {
             'map_url': settings.BASE_MAP,
-            'bbox': json.dumps(settings.MAP_BOUNDING_BOX),
```

Comment:
Are you sure that you want to remove this? You are outputting now python maps/lists directly into the template. This will usually be the [string value](https://docs.djangoproject.com/en/1.10/ref/templates/language/#variables), which might be the reason that works for regular maps.